### ISSUE TITLE
A wish: bookmarks to file(s)

### ISSUE THREAD

**Issue by ilkant** [bibletime/bibletime#263]
It might be good if BT user could save the bookmarks to file. And file format would be xml or plain text. So he or she could have several bible studies in several bookmark files. One study in one file. A study could be started by importing one bookmark file to BT. And he or she could be able to share a set of bookmarks with his or her friend with a file as email attach. Would it be handful and nice?

**Comment by jaakristioja** [comment]
Yes. [XBEL](http://pyxml.sourceforge.net/topics/xbel/) could be one possible format.

**Comment by ilkant** [comment]
Here is little example of bookmark file. This is not very real, but shows that it can be quite simple. One point is that bookmark may point to range of verses, not only one verse.

<xbel>
    <ref>
        <title>Christmas Gospel</title>
        <bibleref bible="KJV">Luke 2:1-20</bibleref>
    <ref>
        <title>Crucifixion</title>
        <bibleref bible="KJV"></bibleref>
</xbel>

**Comment by jaakristioja** [comment]
This is not valid XBEL. It should instead be something like this:

```xml
<xbel version="1.0">
    <bookmark href="sword:KJV:Luke 2:1-20">
        <title>Christmas Gospel</title>
    </bookmark>
</xbel>
```

But as far as I remember there is no specification for a `sword:` URI scheme at this point, so we might have to figure out something. BibleTime-specific data can be embedded in some metadata elements in the `<bookmark>`, e.g. `<info><metadata owner="bibletime:v3:bookmark">data</metadata></info>`.

**Comment by ilkant** [comment]
I have thought that XBEL. I think that it might be good idea to inherit XBEL another Bible verses oriented language.
